src/app/code/community/Wf/Bananastand/Helper/Data.php

```diff
@@ -143,5 +143,85 @@ public function isDebugMode()
         return Mage::getStoreConfigFlag('banana/general/debug_mode');
     }
 
+    /**
+     * @return json - The contents of the cart as a json string.
+     */
+    public function getCartJson()
+    {
+        $cartItems = Mage::getModel('checkout/cart')->getItems();
+        if (empty($cartItems)) {
+          return "[]";
+        }
+        return json_encode($cartItems->getData());
+    }
+
+    /**
+     * Format a mage order object into JSON for sending to VIA js api
+     * @param Mage_Sales_Model_Order $order
+     * @return JSON
+     */
+    public function jsonifyOrder($order)
```

Comment:
We should move this code to model class. Create a class that is called `Wf_Bananastand_Model_Order` and have a method like:
```php
$feraOrder = Mage::getModel('banana/order')->loadFromMageOrder($order);
$feraOrder->getJson(); // returns JSON string of data ready to be sent to Fera API
```